Add more detail to mobile log-in steps (#25093)
DataDog/documentation: content/en/mobile/_index.md

SAML login requires you to set up and authenticate your SAML provider with Datadog. For SAML IdP-initiated login, refer to the end of this section. To authenticate SAML:

> 1. In the mobile app, select your data center region (for example, US1).

Reviewer: Since we're not adding a screenshot, adding some navigation element to the sentence may help!

```suggestion
1. In the mobile app, select your data center region (for example, US1) on the upper right hand corner.
```

2. Press the log-in button.
3. Click the "Using Single Sign-On (SAML)?" link.
4. Enter your company email and send the email.
5. While on your mobile device, open the email and click on the indicated link.

> 6. Enter your org's SAML credentials, which upon success will reroute to an authenticated session of the Datadog mobile app.

Reviewer: ```suggestion
6. Enter your org's SAML credentials. Upon success, reroutes you to an authenticated session of the Datadog mobile app.
```
